docs/index.md

```diff
@@ -1,6 +1,9 @@
 ---
 layout: default
 ---
+# Vision
+
+The Center for Open-Source Research Software Stewardship and Advancement (CORSA) vision is to become a community of practice that enables pathways for open-source scientific software projects to avail themselves of resources for long-term growth, stewardship, advancement, and innovation.
```

Comment:
This seems like a mission rather than a vision to me, but I know different people read these differently.

Reply:
I will commit this and if you'd like to edit that is fine. Thanks!